### ISSUE TITLE
Make sure /var/www/html directory exists before creating symlink

### ISSUE THREAD

**Issue by johanek** [johanek/johanek-redmine#42]
```
Error: Could not set 'link' on ensure: No such file or directory - /var/www/html at 15:/tmp/kitchen/modules/redmine/manifests/config.pp
```

**Comment by UnrealQuester** [comment]
The main problem is the default webroot /var/www/html. This directory is the default docroot for Redhat based systems. Maybe we should set it to apache::docroot by default. See #44

**Comment by johanek** [comment]
Do we even need the symlink? I chose to do it that way because I like to see all the apps available from the apache webroot, and I had this idea of putting different versions in /usr/src/redmine-<version> and symlinking the correct one.

The app could be served from anywhere, so we could just use `$redmine::install_dir` everywhere maybe?

If that's not viable then using `$apache::docroot` sounds good.

**Comment by UnrealQuester** [comment]
In this case we should move it back to 3.0 for sure. Apache does not care where the docroot is. Not sure what the impact for #33 would be. But removing code without removing feature is always good. What should happen with the configurations that set install_dir and webroot to different paths?

**Comment by johanek** [comment]
It's not a huge win, so I wouldn't want to do this is it makes the subdir functionality harder.

Thinking about this more, this is a bug that needs fixing no matter what, so if we change to use `$apache::docroot` for 2.2 we should just leave it that way.

**Comment by UnrealQuester** [comment]
Closed by #44
